partner_event/wizard/res_partner_register_event.py

```diff
@@ -27,10 +27,9 @@ def _prepare_registration(self, partner):
     @api.multi
     def button_register(self):
         registration_obj = self.env['event.registration']
-        partner_obj = self.env['res.partner']
         errors = []
-        for partner_id in self.env.context.get('active_ids', []):
-            partner = partner_obj.browse(partner_id)
+        for partner in self.env['res.partner'].browse(
+                self.env.context.get('active_ids', [])):
```

Comment:
Incorrect indentation

Reply:
It's accepted in PEP8 for `if` blocks, so I guess for `for` blocks too.

Reply:
Ah, sorry, my bad. I though it was the first line inside the block.